scripts/pages/main-menu/main-menu.ts

```diff
@@ -287,11 +290,17 @@ class MainMenu {
 		this.panels.image.visible = !useVideo;
 		this.panels.image.SetReadyForDisplay(!useVideo);
 
-		let chapter = GameInterfaceAPI.GetSettingInt('sv_unlockedchapters');
-		if (chapter < 1) chapter = 1;
-		if (chapter > 5) chapter = 5;
+		const chapter = GameInterfaceAPI.GetSettingInt('sv_unlockedchapters');
```

Comment:
This is going to need to be changed to account for the new campaign system, after that's merged. I believe sv_unlockedchapters is going to be outright removed since it's not scoped to any individual campaign.